### PR TITLE
Watchman Revamp

### PR DESCRIPTION
**Pull request by Gyrandola**
## Watchman Revamp

- New armor and armor sprites for the citywatch, credits to INFRAREDBARON
image

- Cudgel changed to Stun Maces (ATTENTION: ITS ONLY A RESKIN, NO ACTUAL STUN EFFECTS)
- Disable stunmaces from being purchaseable.

- Watchman Outfit
<details>

<summary>Details</summary>

image
image

</details>

- Sheriff Outfit (He wears a full cape instead of the half one)
<details>

<summary>Details</summary>

image
image

</details>

### CODE REVIEW

**Comment by Nakalon** [comment]
omg this is so sexy?

File: code/game/objects/items/rogueweapons/melee/special.dm [hunk 1]
```diff
@@ -132,17 +142,7 @@
...
-		target.electrocute_act(5, src)
-		charge -= 33
-		if(charge <= 0)
-			on = FALSE
-			charge = 0
-			update_icon()
-			if(user.a_intent)
-				var/datum/intent/I = user.a_intent
-				if(istype(I))
-					I.afterchange()
```
Comment (Nakalon) [inline code/game/objects/items/rogueweapons/melee/special.dm:145]: wait so stun maces lost their stun entirely for more damage?

Comment (Gyrandola) [reply code/game/objects/items/rogueweapons/melee/special.dm:145]: they have the same values of cudgels now. Its a temporary measure while I think of less overpowered stun effects

Comment (Nakalon) [reply code/game/objects/items/rogueweapons/melee/special.dm:145]: its just cuz I know noble people like to use these to troll ingame

**Comment by NikolayByondPlayer** [comment]
Somehow I get Terry Pratchett vibes

**Comment by Nakalon** [comment]
> Somehow I get Terry Pratchett vibes 

Omg Niko absolutely. I so would like to go full diskworld and make dwarves genderless

**Comment by Keazra** [comment]
Can we have access to the armory like the boggies have access to theirs?

**Comment by Gyrandola** [comment]
Removed some mineable area near the town, added lucratice mining area beyond the Bastion to incentivize traffic. 
Added a secondary tunnel that passes through the mountain (still beyond the bastion).
Should have fixed the runtimes

**Review by Nakalon — APPROVED** [review]
Tried and tested

**Pull request merged**